Update triggered_delivery.md (#7526)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/delivery_types/triggered_delivery.md

Any of these things will prevent a user who has completed the trigger event from receiving the campaign:

- The user completed the exception event before the time delay had fully elapsed.

> - [abort_message logic] ({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/aborting_messages) has been used and the message has aborted based on the abort_message logic/rules.

Reviewer: ```suggestion
- [`abort_message logic`]({{site.baseurl}}/user_guide/personalization_and_dynamic_content/liquid/aborting_messages) was used and the message was aborted based on the `abort_message` logic or rules.
```
